### DISCUSSION TITLE
How many spiking neurons there are in an SNN?

### DISCUSSION THREAD

**Discussion by xiaoxiaochaochao** [fangwei123456/SpikingFlow#191] [Q&A] [answered]
![图片](https://user-images.githubusercontent.com/44129831/160852214-adc2f64f-5b46-47eb-a967-f2ef5adee78f.png)

Sir, the picture above is from SpikingVGG-11. After the first convolution layer, the feature map is 64×224×224 and a IFNode layer follows. Are there 64×224×224 IF neurons?

**Comment by fangwei123456** [comment, marked as answer]
Yes, the shape of spiking neurons are identical with the input after initialization or reset.
